demo/react/doc/product/components/tooltip.mdx

```diff
@@ -0,0 +1,66 @@
+```javascript import
+```
+
+# Tooltip
+
+**Tooltips display informations when users hover over or focus on an element.**
+
+## Demo
+
+```javascript jsx withThemeSwitcher
+() => (
+    <>
+        Un boutton m qui declenche une tooltip default bottom
+    </>
+);
+```
+
+## Placement
+
+Tooltips may be positioned _top_, _bottom_, _left_, or _right_ to the trigger element.
```

Comment:
Tooltips can be placed to 4 positions from the anchor element : _TOP_, _RIGHT_, _BOTTOM_, _LEFT_

Reply:
"can" instead of "may" because may is to vague